### PR TITLE
Filter by tag

### PR DESCRIPTION
**Pull request by orthagonal**

### CODE REVIEW

File: lib/html.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ module.exports = (methods, routes) => {
       </tr>
       ${routes.map(r => `
         <tr>
-          <td>${r.path}</td>
+          <td><a href="${r.path}">${r.path}</a></td>
```
Comment (jgallen23) [inline lib/html.js]: Is that href right?

File: test/table.html [hunk 2]
```diff
@@ -26,14 +26,14 @@ <h1 id="routes">Routes</h1>
       </tr>
       
         <tr>
-          <td>/appian</td>
+          <td><a href="/appian">/appian</a></td>
```
Comment (jgallen23) [inline test/table.html]: Shouldn't this be ?tag=

File: lib/html.js (hunk 1 above)
Comment (orthagonal) [reply lib/html.js]: This looks/works right in my browser, but do we want the server address here as well instead of just relative path?

Comment (jgallen23) [reply lib/html.js]: how does it work?  you are going to a page that doesn't exist

Comment (jgallen23) [reply lib/html.js]: ....

**Pull request merged**